### ISSUE TITLE
Add typescript examples

### ISSUE THREAD

**Issue by gavr123456789** [clayrisser/react-gtk#26]
In main now I see only js, in refactor branch there are one example on C heh.  
  
P.S. cant wait to try this out.   
Will it be possible to use mobx with it?

**Comment by clayrisser** [comment]
Yes it will be possible. Sorry for the long long wait, but we will have a stable version of this ready by Christmas 2023.

**Comment by gavr123456789** [comment]
Nice to hear

**Comment by clayrisser** [comment]
@gavr123456789 I don't know why I said 2024, I meant 2023. It's coming very soon!!!
